src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java

```diff
@@ -862,4 +863,6 @@ public interface PortalGinInjector extends Ginjector {
   FollowingPagePresenter getFollowingPagePresenter();
 
   ColumnModelsEditorWidget getColumnModelsEditorWidget();
+
+  EntityTypeIcon getEntityTypeIcon();
```

Comment:
`EntityTypeIcon` can be retrieved through PortalGinInjector (and thus may be easily mocked in test)